### ISSUE TITLE
(suggestion) emutool remember FTP info

### ISSUE THREAD

**Issue by GizmoTheGreen** [XorTroll/emuiibo#100]
perhaps even default port to 5000 since thats what switch ftp uses?
keep IP in some ini file maybe?

**Comment by impeeza** [comment]
What you propose, is what the EmuTool remember the last FTP address and port used, and in default the port 5000 is filled at the port box?

**Comment by GizmoTheGreen** [comment]
Yes exactly. Ftpd on switch has port 5000 by default. I'm unsure if sysftpd is the same but I assume so.

**Comment by impeeza** [comment]
Yes, the three Versions of ftp daemons for switch (and 3ds) i have triyed uses 5000 port
